### ISSUE TITLE
Saka Key does not work on some pages

### ISSUE THREAD

**Issue by joefiesta** [lusakasa/saka-key#94]
Every time I have a problem on a page with Saka Key, it suddently occurs to me:  Oh, Saka Key does not work on certain pages.  I read that somewhere, but I'll be darned if I can find it now.

It would be nice if

1.   it was  documented.  (If it is, sorry, I can't find it.)

2.   the documentation included exactly which pages you are talking about.  It is not just  about:addons.  Nor it just all the about: xxxxx   pages.  

And, then, of course, my questions:

1.  Which pages are known problems for Saka Key?

2.  Does this happen only in Firefox?  From much of discussion I have read on Github, there are some (many?) differences between Firefox and other browsers in terms of the code used to implementeSaka Key.  

3.  On these pages, all the built-in Firefox shortcuts (for example CTRL+W to close the tab) work fine.  Is there a reason why Saka Key implementation can not simply invoke the built-in shortcuts?

**Comment by sudoyang** [comment]
Agree. On the firefox's addon page, it does not work. Is it possible to make saka works on almost all pages?

**Comment by eejdoowad** [comment]
Built-in browser pages and Chrome Webstore/Firefox Marketplace pages are where Saka Key doesn't work. The exact list depends on the browser. Feel free to post a comprehensive list if you come up with one, and I'll add it to the handbook. 

It is possible to change the Icon displayed in the toolbar if Saka Key cannot load into a page. This will be part of the fix for #22.

There are no workarounds to get Saka Key to work on all pages. Read other issues for more info.

**Comment by fluffymadness** [comment]
Is it possible to somehow get it to work on New Tab pages ?

**Comment by 3v1n0** [comment]
I guess a new API should be requested to control browser even when the current tab isn't a web page on which we can act...

For example is quite annoying not be able to open, (close but this is less important) or switch a tab containing system content.. So for example when i'm cycling through tabs, this stops working when I select a system one.
